### ISSUE TITLE
[Question] when is next release?

### ISSUE THREAD

**Issue by AndreasAugustin** [awslabs/aws-deployment-framework#443]
Hi,

first thanks for your work. Highly appreciated! The framework is currently helping a lot.

Just a short Question. Is there any plan for a new release?
There hasn't been a release since approx. 1 year.

The current `master` contains several bug fixes (e.q. use of custom pipeline images with a different tag like `latest`).
Also there are many nice to have features within current `master`.

If there is no way to get a release soon, is there a different way to update then using SAR?

Thanks a lot!

**Comment by Nr18** [comment]
@AndreasAugustin as far as I understood the team is preparing a new release, see: https://github.com/awslabs/aws-deployment-framework/milestone/10 but I do not have a date when this will happen.

**Comment by sbkok** [comment]
Hi @AndreasAugustin,

We are working on two releases at the moment.
A hotfix release to address issue #440, this release will only contain a fix to solve the issue that currently affects 3.1.2 installations.

The next minor release will be 3.2.0 as @Nr18 linked to above.
This release will include all the fixes and features you are referring to.
However, some of the pull requests created a conflict situation that had to be resolved first.
I have resolved most issues locally and I am working to get those changes pushed back into the repository.
My guess would be in the next couple of weeks, unless we run into new issues while testing the release candidate of course.

Apologies for the delayed release, one of the issues that we ran into took quite some time to debug.
Thanks for your patience!

Best regards,
Simon
